frontend/util/multi-client-select.component.tsx

```diff
@@ -0,0 +1,77 @@
+import { maybe } from "kremling";
+import React from "react";
+import SingleClientSearchInput from "../client-search/single-client/single-client-search-input.component";
+import CUProgramInputs from "../programs-and-services/cu-program-inputs.component";
+import { SingleClient } from "../view-edit-client/view-client.component";
+import CloseIconButton from "./close-icon-button.component";
+
+const MultiClientSelect = React.forwardRef<
+  MultiClientSelectRef,
+  MultiClientSelectProps
+>((props, ref) => {
+  const [clients, setClients] = React.useState(props.initialClients);
+  const [showEmptyRow, setShowEmptyRow] = React.useState(
+    props.initialClients.length === 0
+  );
+  React.useImperativeHandle(ref, () => ({
+    getClients() {
+      return clients;
+    },
+  }));
+
+  const removeClientSelect = () => {
+    let newArr = [...clients];
+    newArr.pop();
```

Comment:
This looks like it will always remove the last client from the array, instead of the client that is next to the X that was pressed. We should remove the client for which the X was pressed.